eng/pipelines/checkout-windows-task.yml

```diff
@@ -4,6 +4,7 @@ steps:
 
   - script: |
       @echo on
+      set PATH=$(Agent.HomeDirectory)\externals\git\cmd;"%PATH%
```

Comment:
It looks like there's an unmatched `"` here. I wasn't sure if that was deliberate.

Reply:
This is what I get for typing in the web editor.